src/site/components/preview/preview.11tydata.js

```diff
@@ -4,7 +4,9 @@ const { basename, dirname, join } = require('node:path')
 
 const modukFrontendLibPath = dirname(require.resolve('@moduk/frontend'))
 const paths = glob.sync(join(modukFrontendLibPath, 'nunjucks/**/__examples__/*.njk'))
-const examples = paths.map((path) => {
+const localExamples = glob.sync(join(__dirname, '../..', '/_includes/moduk/components/**/__examples__/*.njk'))
```

Comment:
There was a bug on the pages that uses the `/_includes/moduk/components/**/__examples__/*.njk` files It was not generating a preview or embed page.